src/nnet3bin/nnet3-egs-augment-image.cc

```diff
@@ -304,6 +333,18 @@ int main(int argc, char *argv[]) {
       po.PrintUsage();
       exit(1);
     }
+    
```

Comment:
There is a trailing space here. Please fix here and other places and configure your editor to disallow trailing spaces.